src/PlanetData.js

```diff
@@ -98,5 +151,46 @@ PlanetData.prototype = {
 
   getSector: function(x, y) {
     return this.sectors[x][y];
+  },
+
+  //this function is used to return a height at a given point
+  //not completely accurate because river valleys are computed after the fact
+  heightmap: function(x, y) {
+    //this block initializes variables needed for the loop
+    var lh = 0; //local hight//height of individual octave
+    var p = this.frequency;
+    var s = 1 / 6; //scaleing factor for LOD
+    var px = x, py = y;
+    var a = 0.75 + 0.15 * noise.simplex2(px * p, py * p); //Use noise to initilize amplitude in order to create more variation
+    var h = 0.0; //intial total height value
+    var r = [1.212, 0.656 - this.randEffect * 0.3, -0.856 + this.randEffect * 0.3, 1.537]; //rotation matrix for adding interesting effects to frequency calculation

```

Comment:
A few typos in the comments in this section
Line 160: `hight` -> `height` (I think? otherwise I don't know what hight means)
Line 162: `scaleing` -> `scaling`
Line 165: `intial` -> `initial`

Also I think these variables should be given longer names.